packages/oauth/src/install-provider-options.ts

```diff
@@ -2,17 +2,95 @@ import { Logger, LogLevel } from '@slack/logger';
 import { WebClientOptions } from '@slack/web-api';
 import { StateStore } from './state-stores';
 import { InstallationStore } from './stores';
+import { InstallURLOptions } from './install-url-options';
 
 export interface InstallProviderOptions {
+
+  /**
+   * Client ID, which can be found at https://api.slack.com/apps
+   */
   clientId: string;
+
+  /**
+   * Client Secret, which can be found at https://api.slack.com/apps
+   */
   clientSecret: string;
+
+  /**
+   * Manages installation data, which can be called by both the OAuth flow and authorize() in event handling
+   */
+  installationStore?: InstallationStore; // default MemoryInstallationStore
+
+  /**
+   * The slack.com authorize URL
+   */
+  authorizationUrl?: string;
+
+  /**
+   * Manages state parameter values in the OAuth flow
+   */
   stateStore?: StateStore; // default ClearStateStore
+
+  /**
+   * The secret value used for generating the state parameter value
+   */
   stateSecret?: string; // required with default ClearStateStore
+
+  /**
+   * handleCallback() verifies the state parameter if true (default: true)
+   */
   stateVerification?: boolean; // default true, disables state verification when false
-  installationStore?: InstallationStore; // default MemoryInstallationStore
+
+  /**
+   * handleCallback() skips checking browser cookies if true (default: false)
+   * Enabling this option is not really recommended.
+   * This is supposed to be used only for backward-compatibility with v2.4 and olders.
+   */
+  legacyStateVerification?: boolean;
+
+  /**
+   * The cookie name used for setting state parameter value in cookies
+   */
+  stateCookieName?: string;
+
+  /**
+   * The expinary time for the state parameter value in cookies
+   */
+  stateCookieExpirationSeconds?: number;
+
+  /**
+   * The function for rendering the web page for the install path URL
+   */
+  renderHtmlForInstallPath?: (url: string) => string;
+
+  /**
+   * The install path web page rendering will be skipped if true (default: false)
+   */
+  directInstall?: boolean; // default false, disables rendering "Add to Slack" page for /slack/install when true
+
+  /**
+   * The defalt is "v2" ("v1" is for "classic" apps)
```

Comment:
```suggestion
   * The default is "v2" ("v1" is for "classic" apps)
```

Reply:
What specifically does this auth version refer to? Not the OAuth version, right? Is it the authentication procedure prior to introducing this state parameter verification built in mechanism in this PR?

Reply:
OK, thanks for clarifying, good to know. Not sure if it's appropriate to keep these links as breadcrumbs for the comment? Just for future maintainers who may not understand the difference between the versions or even which version this is referring to. Maybe we could have a comment like:

> * The default is "v2" (a.k.a. granular bot permissions), different from "v1" (a.k.a. "classic apps"). More details here: https://medium.com/slack-developer-blog/more-precision-less-restrictions-a3550006f9c3

Reply:
@filmaj Sounds good. I've updated the comments on authVersion.